### PR TITLE
feat(launcher): allow users with `Permission.READ` or above to unzip a study

### LINKED ISSUE #1732: [ANT-849] Change restrictions on POST v1/tasks API
## Description
The AntaresWeb API POST request “v1/tasks” required being admin to be executed correctly (and return results). In our opinion, this query is the only one allowing us to know if the unzipping tasks are still in progress or finished after running a simulation. This is important information when scripting calculation sequences.
 
If there is no logical reason to have to be an admin, can this restriction be lifted?

🚨 Attention: The bug fix of this issue cannot be addressed immediately as it depends on the feature #1124 that introduces a breaking change at the database level. A database schema migration is required to proceed. Please note that this fix will be scheduled for release v2.16, rather than v2.15.5.

### PR DESCRIPTION
**Pull request by MartinBelthle**
I've deployed this branch on staging environment to test with different users and it works fine :)

In this PR, I allowed users with `Permission.READ` or above to unzip a study (previously `WRITE`).
I did it for 2 reasons:
1) I needed users with the `Permission.RUN` to do so for my development.
 2) I do not consider unzipping to be tempering. Therefore I find this more logical.

### CODE REVIEW

**Comment by ghost** [comment]
### Features

*  **launcher:** unzipping task raised by the user who launched the study ([c1467ec](https://github.com/AntaresSimulatorTeam/AntaREST/commit/c1467ec8a550bf472ea9f22d0dcf59ded1c2b938))


### Bug Fixes

*  **service:** allow unzipping with permission read ([f792496](https://github.com/AntaresSimulatorTeam/AntaREST/commit/f792496c601b266205fa4ffea465512bbd4e1d2e))


### Contributors

<a href="https://github.com/MartinBelthle">MartinBelthle</a>

<details>
<summary>Commit-Lint commands</summary>
<br />

You can trigger Commit-Lint actions by commenting on this PR:
- `@Commit-Lint merge patch` will merge dependabot PR on "patch" versions (X.X.Y - Y change)
- `@Commit-Lint merge minor` will merge dependabot PR on "minor" versions (X.Y.Y - Y change)
- `@Commit-Lint merge major` will merge dependabot PR on "major" versions (Y.Y.Y - Y change)
- `@Commit-Lint merge disable` will desactivate merge dependabot PR
- `@Commit-Lint review` will approve dependabot PR
- `@Commit-Lint stop review` will stop approve dependabot PR



</details>

**Comment by laurent-laporte-pro** [comment]
Can you rephrase the title "unzipping task raised by the user who launched the study". It is not really clear for the end-user.

For instance: Allow Users with READ Permission to Unzip Studies

**Review by laurent-laporte-pro — DISMISSED** [review]

**Comment by laurent-laporte-pro** [comment]
@MartinBelthle Pourrais-tu STP changer le titre de la PR, car je ne le comprends pas ?

**Comment by MartinBelthle** [comment]
@laurent-laporte-pro Je n'ai pas regardé les changements que tu as effectué mais le but de cette PR était le suivant :
- La tâche de dézippage automatique d'un résultat (lancée par l'option `auto_unzip`) doit être lancée par l'utilistaeur qui a lancé l'étude.
- Pour que cela fonctionne, j'ai aussi du changer le Permission mode lors du dézippage d'une étude

**Review by laurent-laporte-pro — APPROVED** [review]
